.github/ISSUE_TEMPLATE/4-release_checklist.md

```diff
@@ -10,47 +10,47 @@ assignees: ''
+- [ ] Review the ["PyGMT Team" page](https://www.pygmt.org/dev/team.html)
 - [ ] Check to ensure that:
+  - [ ] Deprecations and related tests are removed for this version by running `grep --include="*.py" -r vX.Y.Z` from the base of the repository
   - [ ] All tests pass in the ["GMT Legacy Tests" workflow](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_tests_legacy.yaml)
   - [ ] All tests pass in the ["GMT Dev Tests" workflow](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_tests_dev.yaml)
   - [ ] All tests pass in the ["Doctests" workflow](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_doctests.yaml)
-  - [ ] Deprecations and related tests are removed for this version by running `grep --include="*.py" -r 'remove_version="vX.Y.Z"' pygmt` from the base of the repository
-- [ ] Update warnings in `pygmt.show_versions()` as well as notes in [Common installation issues](https://www.pygmt.org/dev/install.html#not-working-transparency)
-      and [Testing your install]((https://www.pygmt.org/dev/install.html#testing-your-install) regarding GMT-Ghostscript incompatibility
+- [ ] Update warnings in `pygmt/_show_versions.py` as well as notes in
+      [Not working transparency](https://www.pygmt.org/dev/install.html#not-working-transparency)
+      regarding GMT-Ghostscript incompatibility
 - [ ] Reserve a DOI on [Zenodo](https://zenodo.org) by clicking on "New Version"
-- [ ] Review the ["PyGMT Team" page](https://www.pygmt.org/dev/team.html)
-- [ ] Finish up 'Changelog entry for v0.x.x' Pull Request:
-  - [ ] Add a new entry in `doc/_static/version_switch.js` for documentation switcher
-  - [ ] Update `CITATION.cff` and BibTeX at https://github.com/GenericMappingTools/pygmt#citing-pygmt
-    - [ ] Update authorship list
-    - [ ] Update DOI (and url for BibTeX)
-    - [ ] Update version
-    - [ ] Update date released
-  - [ ] Add the documentation link `doc/minversions.md`
-  - [ ] Add minimum required version information `doc/minversions.md`
-  - [ ] Copy draft changelog from Release Drafter and edit it to look nice ([see maintainers guide for details](https://www.pygmt.org/dev/maintenance.html#updating-the-changelog))
+- [ ] Finish up the "Changelog entry for v0.x.x" Pull Request (Use the previous changelog PR as a reference)
+- [ ] Run `make codespell` to check common misspellings. If there are any, either fix them or add them to `ignore-words-list` in `pyproject.toml`
+- [ ] Draft the announcement on https://hackmd.io/@pygmt
 
 **Release**:
+
 - [ ] At the [PyGMT release page on GitHub](https://github.com/GenericMappingTools/pygmt/releases):
   - [ ] Edit the draft release notes with the finalized changelog
   - [ ] Set the tag version and release title to vX.Y.Z
+  - [ ] Make sure "Set as the latests release" and "Create a discussion for this release" are selected
```

Comment:
"Create a discussion for this release" is a new option after we enable the project "Discussions". See https://github.com/GenericMappingTools/pygmt/discussions/3426 for an example.

Reply:
I don't think we should enable discussions for this repo, best to keep everything on the forum to avoid having too many places to monitor.

Reply:
Are you suggesting (1) Disabling announcements for releases in the "Discussions" page or (2) Disabling the entire ["Discussions" page](https://github.com/GenericMappingTools/pygmt/discussions)?

I'm OK with (1). I prefer to keep the "Discussions" page for non-issue discussions. For example, we may need to discuss the expected release date for v0.14.0, but I don't want to open an issue for it; some issues may be better moved for "Discussions" (e.g., https://github.com/GenericMappingTools/pygmt/issues/3398).

Reply:
> Are you suggesting (1) Disabling announcements for releases in the "Discussions" page or (2) Disabling the entire ["Discussions" page](https://github.com/GenericMappingTools/pygmt/discussions)?

I'd prefer (2) actually - disable a PyGMT-specific discussion page. Less things to keep track of :slightly_smiling_face:

> some technical discussions may be better moved to "Discussions" (e.g., #3398).

Issues/discussions like that are rather rare, but I see your point. I've been thinking if we can convert the GMT repo's discussion at https://github.com/GenericMappingTools/gmt/discussions into an organization-wide discussion (see https://docs.github.com/en/organizations/managing-organization-settings/enabling-or-disabling-github-discussions-for-an-organization), and maybe have a `pygmt` specific category there?

Reply:
The entry is removed in 0708027cfb79507135520205de42862238b0a090.

> I've been thinking if we can convert the GMT repo's discussion at https://github.com/GenericMappingTools/gmt/discussions into an organization-wide discussion (see https://docs.github.com/en/organizations/managing-organization-settings/enabling-or-disabling-github-discussions-for-an-organization), and maybe have a `pygmt` specific category there? I don't have the permissions to do so though.

It seems impossible to subscribe to a specific category. I'm also unsure who will be notified when a discussion is created in the "pygmt" category.